src/main/kotlin/us/timinc/mc/cobblemon/spawnnotification/SpawnNotification.kt

```diff
@@ -35,7 +39,15 @@ object SpawnNotification : ModInitializer {
 
             broadcastSpawn(evt)
             if (config.playShinySound && pokemon.shiny) {
-                playShinySound(evt.ctx.world, evt.ctx.position)
+                val broadcastRange = config.broadcastRange
+                if (broadcastRange > 0) {
```

Comment:
I'm not very happy to be doing two identical checks about broadcastRange. Should probably break it into a util function for DRY reasons.